bepasty_cli/cli.py

```diff
@@ -16,7 +16,7 @@
 import codecs
 import warnings
 try:
-    from ConfigParser import SafeConfigParser as ConfigParser, Error as ConfigParserError
+    from configparser import SafeConfigParser as ConfigParser, Error as ConfigParserError
 except ImportError:
     # Python 3
     from configparser import ConfigParser, Error as ConfigParserError
```

Comment:
that doesn't look correct.

the upper block is meant for python 2, so it should be `ConfigParser`.

if that import fails (e.g. because we are on py3), it is retried in the lower except block.